dask/array/chunk.py

```diff
@@ -265,6 +265,21 @@ def arange(start, stop, step, length, dtype, like=None):
     return res[:-1] if len(res) > length else res
 
 
+def linspace(start, stop, num, endpoint=True, dtype=None):
+    from .core import Array
+
+    if isinstance(start, Array):
+        start = start.compute()
+
+    if isinstance(stop, Array):
+        stop = stop.compute()
+
+    if isinstance(num, Array):
+        num = num.compute()
```

Comment:
According to both [NumPy](https://numpy.org/doc/stable/reference/generated/numpy.linspace.html) and [Dask](https://docs.dask.org/en/latest/array-api.html#dask.array.linspace) docs, `num` is an `int`, not an array-like/Dask array, so we can remove this.

Reply:
Done! Again, thanks for the feedback.